### ISSUE TITLE
MissingMethodException on local execution

### ISSUE THREAD

**Issue by ds1709** [graphql-dotnet/graphql-client#340]
### Package info
Package: GraphQL.Client.LocalExecution
Version: 3.2.3
### Description
`MissingMethodException` thrown when call `SendQueryAsync`.
### Code to reproduce
```csharp
var sourceSchema = Schema.For(@"
    type QT {
        idField: ID
        intField: Int
        floatField: Float
        stringField: String
    }");

var client = GraphQLLocalExecutionClient.New(sourceSchema, new NewtonsoftJsonSerializer());
var request = new GraphQLRequest { Query = "query { __schema { types { name } } }" };
var res = await client.SendQueryAsync<JToken>(request);
```
### Expected
Query successfully executed.
### Actual
MissingMethodException: Method not found: "GraphQL.Inputs GraphQL.StringExtensions.ToInputs(Newtonsoft.Json.Linq.JObject)".

**Comment by ds1709** [comment]
Why package `GraphQL.Client.LocalExecution.3.2.3` depends on `GraphQL >= 2.4.0` then? It shoud be stricted by version `2.4.0` in nuspec file (`[2.4,3)` instead of `2.4.0`).

**Comment by sungam3r** [comment]
In general, yes, but people don't always use the upper version limit of the used packages. Ping @rose-a

**Comment by rose-a** [comment]
I agree this should be updated. Unfortunately I'm currently quite short on time. A PR is of course very welcome.
